src/index.html

```diff
@@ -5,9 +5,61 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="styles/styles.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body class="body">
+    <div class="card" data-qa="card">
+
+      <img class="card__image"
+      src="images/imac.jpeg" alt="APPLE">
```

Comment:
[Formatting](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#html-formatting)
Also it can seems that it is a real apple on image 🍏 give more descriptive `alt`